examples/lookup.php

```diff
@@ -7,8 +7,8 @@
 $factory = new Clue\React\Mdns\Factory();
 $mdns = $factory->createResolver();
 
-$mdns->resolve($name)->then('e', 'e');
-
-function e($v) {
+$mdns->resolve($name)->then(function ($v) {
     echo $v . PHP_EOL;
```

Comment:
This is what this looks like in the readme:

```suggestion
$mdns->resolve($name)->then(function ($ip) {
    echo 'Found: ' . $ip . PHP_EOL;
```

Reply:
Okay, I oriented me on the function below (which is now gone since my changeset), but in any case to have the same example as the Quickstart example is better! Changed it right away. 👍